app/celery/scheduled_tasks.py

```diff
@@ -374,3 +379,70 @@ def delete_old_records_from_events_table():
     current_app.logger.info(f"Deleted {deleted_count} historical events from before {delete_events_before}.")
 
     db.session.commit()
+
+
+@notify_celery.task(name="zendesk-new-email-branding-report")
+def zendesk_new_email_branding_report():
+    previous_weekday = datetime.today().date() - timedelta(days=1)
+
+    # If yesterday is a Saturday or Sunday, adjust back to the Friday
+    if previous_weekday.isoweekday() in {6, 7}:
+        previous_weekday -= timedelta(days=(previous_weekday.isoweekday() - 5))
+
+    new_email_brands = (
+        EmailBranding.query.join(Organisation, isouter=True).filter(EmailBranding.created_at >= previous_weekday).all()
+    )
+
+    current_app.logger.info(f"{len(new_email_brands)} new email brands to review since {previous_weekday}.")
+
+    if not new_email_brands:
+        return
+
+    brands_by_organisation = defaultdict(list)
+    brands_with_no_organisation = []
+    for new_brand in new_email_brands:
+        if not new_brand.organisations:
+            brands_with_no_organisation.append(new_brand)
+
+        else:
+            for organisation in new_brand.organisations:
+                brands_by_organisation[organisation].append(new_brand)
+
+    template = jinja2.Template(
+        textwrap.dedent(
+            """
+            <p>There are new email brands to review since {{ yesterday }}.</p><hr>
+            {%- if brands_by_organisation -%}
```

Comment:
Really minor, but this `if` statement could be removed since if `brands_by_organisation` is empty the `for` loop won't run